include/multiverso/updater/updater.h

```diff
@@ -69,6 +69,47 @@ struct UpdateOption {
   InternalType data_[kSize];
 };
 
+struct GeneralOption {
```

Comment:
If you only use this for get user-specific worker id to make pipeline work, maybe we can implemented with some trick without this extra class. May need some discussion.